x-pack/plugins/profiling/public/services.ts

```diff
@@ -16,14 +18,14 @@ export interface Services {
     timeFrom: number,
     timeTo: number,
     n: number
-  ) => Promise<any[] | HttpFetchError>;
+  ) => Promise<TopNSamples>;
   fetchElasticFlamechart: (
     index: string,
     projectID: number,
     timeFrom: number,
     timeTo: number,
     n: number
-  ) => Promise<any[] | HttpFetchError>;
+  ) => Promise<ElasticFlameGraph>;
```

Comment:
I've removed the Error type. It was not handled and ideally we have a better way to deal with errors.